### PR TITLE
Reduce callers of createEditorInput when not needed (#121098)

### PR DESCRIPTION
**Pull request by bpasero**
Part of #121098

### CODE REVIEW

File: src/vs/workbench/contrib/files/browser/fileCommands.ts [hunk 1]
```diff
@@ -682,9 +680,8 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 		if (typeof args?.viewType === 'string') {
 			const editorGroupsService = accessor.get(IEditorGroupsService);
 
-			const textInput = editorService.createEditorInput({ options: { pinned: true } });
 			const group = editorGroupsService.activeGroup;
-			await editorService.openEditor(textInput, { override: args.viewType, pinned: true }, group);
+			await editorService.openEditor({ options: { override: args.viewType, pinned: true } }, group);
```
Comment (bpasero) [inline src/vs/workbench/contrib/files/browser/fileCommands.ts:684]: I am not entire sure this one works?

**Comment by lramos15** [comment]
From what I'm seeing here it looks like you're putting the options in the resource input you're passing in. I believe the options need to be separate from the input so you would want `openEditor({resource: uri }, options, group)`

File: src/vs/workbench/contrib/files/browser/fileCommands.ts (hunk 1 above)
Comment (lramos15) [reply src/vs/workbench/contrib/files/browser/fileCommands.ts:684]: I'm not sure as well. I don't remember when that code gets called. It looks untitled file specific?

**Comment by bpasero** [comment]
@lramos15 the idea is to explicitly end up in this method overload that does not require to pass in any editor input. My understanding is that the override works fine then:

https://github.com/microsoft/vscode/blob/7164efa38e554e45a17437c4171062ea79c56cc8/src/vs/workbench/services/editor/common/editorService.ts#L183-L183

**Review by lramos15 — APPROVED** [review]
No issues from me, seems like they were just calling create on the untyped input anyways.

**Pull request merged**